.golangci.yml

```diff
@@ -1,4 +1,6 @@
 run:
+  timeout: 10m
+
```

Comment:
Dangling `run:`  did not look like a valid yaml so I decided to move timeout config from Makefile